### PR TITLE
scrollbar visibilty issues in nav-container

### PR DESCRIPTION
**Pull request by psingh587**
- From my personal experience, I feel the scroll-bar in nav-container is very light. It sometimes is not even visible and also the width is pretty small which makes it less clickable. 
Before Changes:-

Screenshot from 2020-03-29 17-12-05

After changes:-
#Also it changes color on hovering. I have made the hover color a bit darker than the scrollbar-color
Screenshot from 2020-03-29 17-12-15

### CODE REVIEW

File: antora-ui-camel/src/css/nav.css [hunk 1]
```diff
@@ -89,11 +89,15 @@ html.is-clipped--nav {
 }
 
 .nav-panel-menu::-webkit-scrollbar {
-  width: 0.25rem;
+  width: 0.29rem;
 }
 
 .nav-panel-menu::-webkit-scrollbar-thumb {
-  background-color: var(--nav-border-color);
+  background-color: #e3dcdc;
```
Comment (zregvart) [inline antora-ui-camel/src/css/nav.css]: Change the variable value, don't hard code values.

File: antora-ui-camel/src/css/nav.css [hunk 2]
```diff
@@ -89,11 +89,15 @@ html.is-clipped--nav {
...
+}
+
+.nav-panel-menu::-webkit-scrollbar-thumb:hover {
+  background-color: #afa7a7;
```
Comment (zregvart) [inline antora-ui-camel/src/css/nav.css]: Use a variable here.

**Comment by psingh587** [comment]
@zregvart  Made the changes.

File: antora-ui-camel/src/css/vars.css [hunk 3]
```diff
@@ -6,6 +6,7 @@
   --color-smoke-50: #f5f5f5;
   --color-smoke-70: #f0f0f0;
   --color-smoke-90: #e1e1e1;
+  --color-smoke-100: #e3dcdc;
```
Comment (zregvart) [inline antora-ui-camel/src/css/vars.css:9]: Why introduce yet another color, could we not use `--color-smoke-90`?

File: antora-ui-camel/src/img/close-outline.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g data-name="Layer 2"><g data-name="close"><rect width="24" height="24" transform="rotate(180 12 12)" opacity="0"/><path d="M13.41 12l4.3-4.29a1 1 0 1 0-1.42-1.42L12 10.59l-4.29-4.3a1 1 0 0 0-1.42 1.42l4.3 4.29-4.3 4.29a1 1 0 0 0 0 1.42 1 1 0 0 0 1.42 0l4.29-4.3 4.29 4.3a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42z"/></g></g></svg>
```
Comment (zregvart) [inline antora-ui-camel/src/img/close-outline.svg]: Unrelated change

Comment (psingh587) [reply antora-ui-camel/src/img/close-outline.svg]: Yeah I have already removed this.

File: antora-ui-camel/src/css/vars.css (hunk 3 above)
Comment (psingh587) [reply antora-ui-camel/src/css/vars.css:9]: I was looking for a darker shade so that it is visible even in the brightest screens. I guess --colot-smoke-90 would work too.

**Comment by psingh587** [comment]
> Needs some improvement.

Made the changes @zregvart

**Comment by zregvart** [comment]
Thanks!
